requirements/deploy.txt

```diff
@@ -234,7 +236,7 @@ markdown==3.4.4
     # via -r requirements/pip.txt
 matplotlib-inline==0.1.6
     # via ipython
-newrelic==8.11.0
+newrelic==9.0.0
```

Comment:
We may want to take a look at the changelog here

Reply:
https://docs.newrelic.com/docs/release-notes/agent-release-notes/python-release-notes/python-agent-90000/

Nothing to worry about.